credsweeper/rules/config.yaml

```diff
@@ -644,7 +659,7 @@
   confidence: moderate
   type: pattern
   values:
-    - (?<![0-9A-Za-z_+-])(?P<value>SK[0-9a-fA-F]{32})(?![=0-9A-Za-z_+-])
+    - (?<![0-9A-Za-z_+-])(?P<value>SK[0-9A-Fa-f]{32})(?![=0-9A-Za-z_+-])
```

Comment:
Do they have difference? Did you changed it just for beautify?

Reply:
Yes, i try to unify the pattern with all rules to quick search them in future
The order relates ASCII table.

Reply:
Good idea.